ui-tests/src/test/java/io/jenkins/plugins/analysis/warnings/UiTest.java

```diff
@@ -251,6 +254,43 @@ protected void verifyCheckStyle(final Build build) {
                 .hasErrorMessages("Can't create fingerprints for some files:");
     }
 
+    protected void verifyPep8(final Build build) {
+        verifyPep8(build, 1);
+    }
+
+    protected void verifyPep8(final Build build, final int referenceBuild) {
+        build.open();
+        AnalysisSummary pep8 = new AnalysisSummary(build, PEP8_ID);
+        assertThat(pep8).isDisplayed()
+                .hasTitleText(PEP8_NAME + ": 8 warnings")
+                .hasReferenceBuild(referenceBuild)
+                .hasInfoType(InfoType.ERROR);
+
+        AnalysisResult pep8Details = verifyPep8Details(pep8);
+
+        pep8Details.openTab(Tab.ISSUES);
+        IssuesDetailsTable issuesTable = pep8Details.openIssuesTable();
+        assertThat(issuesTable).hasSize(8);
+
+        long normalIssueCount = issuesTable.getTableRows().stream()
+                .map(row -> row.getAs(IssuesTableRow.class).getSeverity())
+                .filter(severity -> severity.equals("Normal")).count();
+
+        long lowIssueCount = issuesTable.getTableRows().stream()
+                .map(row -> row.getAs(IssuesTableRow.class).getSeverity())
+                .filter(severity -> severity.equals("Low")).count();
+
+        assertThat(normalIssueCount).isEqualTo(6);
+        assertThat(lowIssueCount).isEqualTo(2);
+    }
```

Comment:
Wenn ich auch hier, wie bei den andern verify..() methoden die InfoView vergleichen möchte, dann kann leider die info page nicht gefunden werden. (Obwohl es manuell per URL eingeben klappt)

Wissen Sie eventuell warum das so ist?


`assertThat(openInfoView(build, PEP8_ID))
                .hasInfoMessages("-> found 1 file",
                        "-> found 8 issues (skipped 0 duplicates)",
                        "Issues delta (vs. reference build): outstanding: 0, new: 8, fixed: 0")
                .hasErrorMessages("Can't create fingerprints for some files:");`

Reply:
Ich glaube, Sie müssen `build.open()` vorher ausführen. (Sinnvollerweise sollte das vielleicht in der `openInfoView` Methode selbst gemacht werden, dann muss man das nicht jedesmal selbst davor ausführen)

Reply:
Ohh mann... das ist ärgerlich. Ich hab einfach vergessen, den build.open() Befehl mit zu ziehen und es ist mir nicht aufgefallen.
Vielen Dank für den Hinweis :)

Ich hab den `build.open() ` Befehl mit reingezogen (Und zwar schon einen Schritt vorher im Konstruktor der Summary, weil das da ja eigentlich auch schon notwendig ist.)